### PR TITLE
The hint of add new places

### PR DESCRIPTION
**Pull request by zoeszhen**

### CODE REVIEW

**Comment by Kapoor-uh** [comment]
We can make 'UPDATE' typo error correct and make pull request again so as to avoid confusion.
Also we can add same comment which is in string name= "places_how_to_create" inspite of "Hint: Zoom-in and zoom-out of the map will help you pin the location." so as to maintain uniformity in application

**Comment by newa34** [comment]
How about just to keep hint on map and remove from View  Places. There is repetition same information.

**Comment by Kapoor-uh** [comment]
Yes that's  cool too ..

File: App/src/main/res/anim/add_place_buttons_hide.xml [hunk 1]
```diff
@@ -6,4 +6,5 @@
         android:fromYDelta="0%"
         android:toYDelta="100%"
         android:duration="300" />
+
```
Comment (ffagerho) [inline App/src/main/res/anim/add_place_buttons_hide.xml]: Nitpick: please avoid no-op changes, they make the pull request more confusing. :)

File: App/src/main/res/anim/add_place_buttons_show.xml [hunk 2]
```diff
@@ -6,4 +6,5 @@
         android:fromYDelta="100%"
         android:toYDelta="0%"
         android:duration="300" />
+
```
Comment (ffagerho) [inline App/src/main/res/anim/add_place_buttons_show.xml]: Nitpick: please avoid no-op changes, they make the pull request more confusing. :)

File: App/src/main/res/values/analytics_strings.xml [hunk 3]
```diff
@@ -40,6 +40,7 @@
     <string translatable="false" name="analytics_label_add_place_overlay_activated">\"Add place\" overlay activated</string>
     <string translatable="false" name="analytics_label_cancel_add_place_button">Cancel \"Add place\" button</string>
     <string translatable="false" name="analytics_label_confirm_add_place_button">Confirm \"Add place\" button</string>
+    <string translatable="false" name="analytics_label_confirm_add_place_hint">You can try</string>
```
Comment (ffagerho) [inline App/src/main/res/values/analytics_strings.xml]: Is this sentence incomplete? Try what?

File: App/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -73,6 +73,7 @@
     <string name="required">Required</string>
     <string name="place_name_already_in_use">Name is already in use</string>
     <string name="add_place">Add place</string>
+    <string name="add_place_hint">Hint: Zoom-in and zoom-out of the map will help you pin the location.</string>
```
Comment (ffagerho) [inline App/src/main/res/values/strings.xml]: I would go for: Hint: You can zoom and pan the map to adjust the location.

**Pull request merged**